Tests/MongoSwiftTests/AsyncTestUtils.swift

```diff
@@ -38,6 +38,16 @@ extension MongoClient {
             return try ServerVersion(versionString)
         }
     }
+
+    /// Determine whether server version and topology requirements for a certain test are met
+    internal func meetsRequirements(_ testRequirement: TestRequirement) throws -> UnmetRequirements? {
```

Comment:
since this is no longer returning a boolean value, I think refactoring it to something like `checkRequirements` may make more sense.

Reply:
done